src/main/java/org/opentripplanner/model/vehicle_position/RealtimeVehiclePosition.java

```diff
@@ -0,0 +1,71 @@
+package org.opentripplanner.model.vehicle_position;
+
+import java.time.Instant;
+import org.opentripplanner.model.StopLocation;
+
+/**
+ * A simplified vehicle position class that supports being output to HTTP. Compatible with
```

Comment:
> `supports being output to HTTP`

If we want to use this in the REST API, we should have a separate API class, but I don't see any usage

Reply:
No, it's left over from the original IBI PR. I will remove.